skyreader/result.py

```diff
@@ -1096,30 +1058,31 @@ def area(vs):
                               ra, ra_plus, np.abs(ra_minus)) + " \n" + \
                           "\t Dec = {0:.2f} + {1:.2f} - {2:.2f}".format(
                               dec, dec_plus, np.abs(dec_minus))
-            log_func(contain_txt)
```

Comment:
I remember there's a negative sign in the bounding box area calculation. I think this is because the order of the contour is reversed, so Green's theorem when applied gives the negative of the area. Maybe this can be fixed in the PR here?

Reply:
The `area()` function was also duplicate. I have moved its definition to the class level at `calculate_area()`. One could rewrite it using numpy vectorization, added a TODO for that.

I am now taking the absolute value of the results where applicable.